pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -293,7 +304,8 @@ func (r *ReconcileNode) reconcileCrashRetention(namespace string, cephCluster ce
 		}
 	} else {
 		logger.Debugf("daysToRetain set to: %d", cephCluster.Spec.CrashCollector.DaysToRetain)
-		op, err := r.createOrUpdateCephCron(cephCluster, cephVersion)
+		mink8sversion := k8sVersion.AtLeast(version.MustParseSemantic(minVersionForCronV1))
+		op, err := r.createOrUpdateCephCron(cephCluster, cephVersion, mink8sversion)
```

Comment:
How about renaming this var everywhere? 
```suggestion
		op, err := r.createOrUpdateCephCron(cephCluster, cephVersion, useCronJobV1)
```

Reply:
done